scripts/app.js

```diff
@@ -1,22 +1,24 @@
-import renderMain from "./components/mainComponent.js";
+import createEle from "./utils/createEle.js";
 import renderBodyMap from "./components/body-map/bodyMap.component.js";
 import renderHeader from "./components/header/header.component.js";
-import dinoOfTheDay from "./components/randoDino/randoDino.component";
+import dinoOfTheDay from "./components/dino-of-the-day/dino-of-the-day.component.js";
 import renderShowcase from "./components/showcase-section/showcase-section.component.js";
 import renderCharts from "./components/chart/dinoDietChart.component.js";
 import renderFooter from "./components/footer/footer.component.js";
 import renderIntroSection from "./components/random-dino-section/intro-section.component.js";
+import renderAllDinos from "./components/renderAllDinos/renderAllDinos.js";
 import "../styles/main.scss";
 
 renderHeader();
-await renderMain();
```

Comment:
In the previous version where JavaScript may take some time to "render" which encourages the use of await syntax. Now that renderMain( ) has been broken down into smaller components, is it correct that the page will load the same speed or even faster than before?

Reply:
It's faster now because renderMain is gone. We only await the renderIntroSection now. The rest runs synchronously. But faster only in milliseconds. 

Cleaner more concise, easier for humans to read. But the speed is essentially the same.